code/app/build.gradle

```diff
@@ -35,4 +34,6 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    implementation 'com.github.dhaval2404:imagepicker:2.1'
```

Comment:
@gokul887 @nidal312 @raquibk Can you please double check if we need these changes in the build.gradle?

Reply:
Dependency is required for a library I am using.

Reply:
Is there no standard library for the choosing images? Are we allowed to use 3rd party libraries for the project?